spec/std/isa/inst/Zaamo/amomax.d.aqrl.yaml

```
@@ -0,0 +1,142 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+
+# WARNING: This file is auto-generated from spec/std/isa/inst/Zaamo/amomax.SIZE.AQRL.layout
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: amomax.d.aqrl
+long_name: Atomic MAX doubleword (acquire) (release) (acquire-release)
+description: |
+  Atomically with acquire ordering with release ordering:
```

This doesn't read well, either. Perhaps:
```suggestion
  Atomically with acquire ordering and release ordering:
```
or better:
```suggestion
  Atomically with acquire and release ordering:
```